islandora_usage_stats.module

```diff
@@ -564,3 +561,12 @@ function islandora_usage_stats_menu_alter(&$items) {
   $items['islandora/object/%islandora_object/datastream/%islandora_datastream/download']['module']
     = "islandora_usage_stats";
 }
+/**
+ * Implements hook_islandora_overview_object_alter().
+ */
+function islandora_usage_stats_collections_islandora_overview_object_alter(AbstractObject &$object, &$output) {
```

Comment:
The `collections` bit in this function name would(/should) cause it to never be matched... `islandora:collectionCModel` is the content model we want to target, yeah? To make a function name, we transform "invalid" characters (`:`, `-`, _etc_) to underscores (and PHP function names are not case sensitive)... So to have this alter happen only on objects with the `islandora:collectionCModel` model, the function should likely be named `islandora_usage_stats_islandora_collectioncmodel_islandora_overview_object_alter`.